src/importexport/musicxml/internal/musicxml/import/musicxmlpart.h

```diff
@@ -21,43 +21,41 @@
  */
 #pragma once
 
-#include "types/fraction.h"
-#include "dom/interval.h"
```

Comment:
The Build Without Qt error seems to suggest that this #include is necessary in `importmusicxmlnoteduration.cpp`.